TYPO3.Flow/Classes/TYPO3/Flow/Object/Proxy/Compiler.php

```diff
@@ -141,6 +141,9 @@ public function getProxyClass($fullClassName)
         if (in_array(substr($fullClassName, 0, 14), $this->blacklistedSubPackages)) {
             return false;
         }
+        if ($classReflection->isFinal() && preg_match('/^\s?\*\s?\@Annotation\s/m', $classReflection->getDocComment()) === 1) {
```

Comment:
See https://github.com/neos/flow-development-collection/pull/644#issuecomment-249910726
Without this, Flow complaints about proxied Annotation classes.
Maybe this could be solved otherwise but a `@Flow\Proxy(false)` annotation on each Annotation class is not a good option IMO